File: swack-sheep/src/main/java/bean/WorkspaceList.java
package bean;

public class WorkspaceList {
	/** ワークスペースID */
	private String workspaceID;
	/** ワークスペース名 **/
	private String workspaceName;
	
	public WorkspaceList(String workspaceID, String workspaceName) {
		this.workspaceID = workspaceID;
		this.workspaceName = workspaceName;
	}
	
	public String getWorkspaceID() {
		return workspaceID;
	}
	public String getWorkspaceName() {
		return workspaceName;
	}
}

File: swack-sheep/src/main/webapp/js/workspaceList.js
/**
 * workspaceListから押されたリンクに関連するworkspaceをServletに送信
 */

function clickWorkspace(num1, num2) {
	console.log(num1);
	document.getElementById("workspaceName").value = num1;
	document.getElementById("workspaceID").value = num2;
    document.getElementById("wSpan1").textContent  = document.getElementById("workspaceName").value;
    document.getElementById("wSpan2").textContent  = document.getElementById("workspaceID").value;
};